src/collections/_documentation/workflow/notifications/index.md

```diff
@@ -3,30 +3,94 @@ title: Notifications
 sidebar_order: 2
 ---
 
-Notifications in Sentry can be routed to many supported [integrations]({%- link _documentation/workflow/integrations/index.md-%}), but by default are aimed at email. Notifications fall into two categories:
+Notifications in Sentry can be routed to many supported integrations, but by default are aimed at email. Notifications fall into two categories: Alerts and Workflow.
 
-Alerts
+## Alerts
 
-: Generated based upon a project’s alert rules. By default this is only when a new issue is seen. [Learn more]({%- link _documentation/workflow/notifications/alerts.md -%}).
+Each time you create a new project, or a new issue occurs, Sentry creates a default alert rule. Alerts are generated based upon a project’s alert rules. You can customize alerts to the needs of an individual or team.
 
-Workflow
+Alerts have a set of configuration and subscription options. By default, Sentry subscribes you to everything, but you can change this via account settings. You can also configure various personal notifications from your [personal notifications settings](https://sentry.io/settings/account/notifications) and any specific project's settings in the Project Settings page. For more details, see the [full documentation on Alerts]({%- link _documentation/workflow/notifications/alerts.md -%}).
 
-: All activity related to user actions such as resolution, comments, and automatic regressions. [Learn more]({%- link _documentation/workflow/notifications/workflow.md -%}).
+## Workflow
 
-Each category has its own set of configuration and subscription options. By default you’ll be subscribed to everything, but this can be changed via your [account settings](https://sentry.io/settings/account/notifications/). Read about [notification management]({%- link _documentation/workflow/notifications/notification-management.md -%}) for more information.
+Workflow relates to all activity involving user actions such as resolution, comments, and automatic regressions.
+
+Workflow has a set of configuration and subscription options. By default, Sentry subscribes you to everything, but you can change this via your account settings. You can also configure various personal notifications from your [personal notifications settings](https://sentry.io/settings/account/notifications) and any specific project's settings in the Project Settings page. For more details, see the [full documentation on Workflow]({%- link _documentation/workflow/notifications/workflow.md -%}).
```

Comment:
What do you think about directly linking to the workflow notification fine tuning (https://sentry.io/settings/account/notifications/workflow/) here? I think it might be more clear since we repeat the same information above but refer to the more general notification settings.